### PR TITLE
Dockerfile

### PR DESCRIPTION
**Pull request by mafrosis**
Updated `requirements.txt` with a couple of missing deps, and broke out the dependencies install into a separate docker image layer.

Feel free to close if you don't like this change :)

### CODE REVIEW

**Comment by mide** [comment]
> I don't know if the keys work into the container anyway.

If you're using something that can do USB Passthrough (Like Docker Toolbox on Mac), you can pass the key through. 

If you're using something that can't (Like Docker for Mac), you can not

**Comment by stevemac007** [comment]
Thanks @mide for the clarification - so we should install the `u2f` flavour in the container to suit all needs.

File: Dockerfile [hunk 1]
```diff
@@ -6,8 +6,11 @@ RUN apk add --update-cache py3-pip ca-certificates py3-certifi py3-lxml\
                            jpeg-dev zlib-dev freetype-dev lcms2-dev openjpeg-dev \
                            tiff-dev tk-dev tcl-dev
 
-ADD . /build/
-RUN pip3 install -e /build/[u2f]
+COPY setup.py README.rst requirements.txt /build/
+RUN pip3 install -r /build/requirements.txt
+
+COPY aws_google_auth /build/aws_google_auth
+RUN pip3 install -e /build
```
Comment (stevemac007) [inline Dockerfile]: Please update to the [u2f] build version.

Comment (mafrosis) [reply Dockerfile]: Yep the removal of `[u2f]` wasn't intentional :)

**Comment by mafrosis** [comment]
ping @stevemac007 - P/R updated and rebased against `master`

**Pull request merged**